lib/human_player.rb

```diff
@@ -2,13 +2,13 @@
 
 class HumanPlayer < Player
 
-  def initialize(player_num, board_array)
+  def initialize(player_num, board)
     @player_num = player_num
-    @board_array = board_array
+    @board = board
```

Comment:
same comment as jason above. Why does the player have to have its own board?

Reply:
^ Addressed above. Boards are now being passed in to the get_next_move methods